server/models/Request.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: 'User'
+  },
+  sitterId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: 'User'
+  },
+  start: {
+    type: Date,
+    required:true
```

Comment:
```suggestion
    required: true
```

Throughout the schema look for areas for proper whitespacing please.